lib/internal/Magento/Framework/Setup/Declaration/Schema/Db/MySQL/Definition/Columns/Integer.php

```diff
@@ -53,7 +54,7 @@ class Integer implements DbDefinitionProcessorInterface
      * Constructor.
      *
      * @param Unsigned $unsigned
-     * @param bool $boolean
+     * @param Boolean $boolean
```

Comment:
Hi @mrtuvn. Since PSR-1 is dead, we need to follow PSR-12 which recommends using a short form of types instead of the full one. Please, check the following link for details 

https://www.php-fig.org/psr/psr-12/#25-keywords-and-types

Reply:
This is not cast type . We need pass Class to that property
In this case is \Magento\Framework\Setup\Declaration\Schema\Db\MySQL\Definition\Columns\Boolean. I use short class is Boolean

Reply:
https://github.com/magento/magento2/blob/c58257cbf1ae2294a783a518cd47012d930674a8/lib/internal/Magento/Framework/Setup/Declaration/Schema/Db/MySQL/Definition/Columns/Integer.php#L65

Reply:
used bool in current magento code base is incorrect

Reply:
Got it, thanks